infra/build_deploy_config.py

```diff
@@ -111,6 +140,12 @@
         val = os.environ[key]
         fp.write(terraform_variable_template.format(name=key, val=val))
 
+    fp.write(terraform_variable_template.format(name="project", val=project))
```

Comment:
I'm not sure you want to be writing the same variable into the file multiple times.  Also, `.format()` can take in all variables at once as a dictionary.

Reply:
We would have to rewrite the same string to file multiple times to populate the TF variables, just like on ln.141 ...

unfortunately env_vars_to_infra is a list and not a dict, otherwise the values could have been added, and the same loop could do all the work....

Reply:
A loop would be nicer but in the end just a nit.  I'll resolve this.